queue/src/test/java/org/killbill/notificationq/TestNotificationQueue.java

```diff
@@ -382,7 +392,7 @@ public boolean apply(final Boolean input) {
      *
      * @throws Exception
      */
-    @Test(groups = "slow")
+    @Test(groups = "slow", enabled=false)
```

Comment:
Are these old tests (why disabled)? Should we keep these around?

Reply:
Absolutely, mistake, will fix.